tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -110,6 +110,30 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+#SDCA multi-function
```

Comment:
@bardliao 2 questions:
1. WHy is the BE ID important in the topology name?
2. Why do you need to make HDMI optional based on the number of HDMI's in sof-hda-generic in the first patch when that you never modify NUM_HDMI for any of the HDMI topologies?

Reply:
> @bardliao 2 questions:
> 
> 1. WHy is the BE ID important in the topology name?

The BE id in the topology should match the BE id exposed by the machine driver.
And BE id could be different between devices. For example, HDMI BE ids depends on how many SDW links are created and is PCH DMIC link is created.

> 2. Why do you need to make HDMI optional based on the number of HDMI's in sof-hda-generic in the first patch when that you never modify NUM_HDMI for any of the HDMI topologies?

I would like to split the sof-hda-generic topology in the end. So, we need to make HDMI optional and remove the HDMI part from the generic sof-hda-generic topologies.